FE/qr-menu/src/components/AdminView/Items.jsx

```diff
@@ -50,23 +49,13 @@ export default function Items() {
   const openModal = (id) => setOpenedModal(id);
   const closeModal = () => setOpenedModal(null);
 
-  const [allAllergens, setAllAllergens] = useState([]);
   const [checkedAllergens, setCheckedAllergens] = useState([]);
 
-  useEffect(() => {
-    const fetchAllergens = async () => {
-      try {
-        const response = await axios.get(
-          'https://qrmenu-asdit.herokuapp.com/api/v1/allergens/'
-        );
-        setAllAllergens(response.data.allergens);
-      } catch (err) {
-        console.log(err.response);
-      }
-    };
-
-    fetchAllergens();
-  }, []);
+  const { data } = useQuery('allergens', () =>
+    fetch('https://qrmenu-asdit.herokuapp.com/api/v1/allergens/').then((res) =>
```

Comment:
we want to use default fetch address and only use something like `/api/v1/...` here